djangocms_attributes_field/widgets.py

```diff
@@ -44,8 +44,8 @@ def _render_row(self, key, value, field_name, key_attrs, val_attrs):
                <label>Value</label>
                <input type="text" class="attributes-value"
                       name="attributes_value[{field_name}]"
-                      value="{value}" {val_attrs}>
-                <a class="delete-attributes-pair deletelink" href="#" title="{remove}"></a>
+                      value="{value}" {val_attrs}><!--
+                --><a class="delete-attributes-pair deletelink" href="#" title="{remove}"></a>
```

Comment:
good catch actually, i'd prefer spaceless, but will have to look up how to use it on plain string